lib/recurly/pager.php

```diff
@@ -123,6 +123,7 @@ private function _loadLinks($response) {
       preg_match_all('/\<([^>]+)\>; rel=\"([^"]+)\"/', $links, $matches);
       if (sizeof($matches) > 2) {
         for ($i = 0; $i < sizeof($matches[1]); $i++) {
+          $matches[1][$i] = str_replace('http://', 'https://', $matches[1][$i]);
```

Comment:
This is a "hack", but since even the tests have an assertion for http:// it seems the recurly api returns this for a long time already, not pointing to https directly.